.github/workflows/release-app.yml

```diff
@@ -148,6 +148,11 @@ jobs:
             pre-build-args: ""
             tauri-args: "--target x86_64-pc-windows-msvc --config src-tauri/tauri.conf.prod.json"
             os_type: "windows"
+          - platform: "ubuntu-22.04"
+            args: "--target x86_64-unknown-linux-gnu --features mkl"
+            target: x86_64-unknown-linux-gnu
+            tauri-args: "--target x86_64-unknown-linux-gnu --config src-tauri/tauri.conf.prod.json"
```

Comment:
did u rebase

Reply:
we dont have --config src-tauri/tauri.conf.prod.json anymore

Reply:
@louis030195 Fixed